lib/shiika_parser/src/lexer.rs

```diff
@@ -325,7 +329,7 @@ impl<'a> Lexer<'a> {
                 break;
             }
         }
-        Token::Separator
```

Comment:
The implementation read_comment returns Token::Newline seems does not have any problem.
But I don't have absolute confidence about this behavior.

If you have any concern about this implementation, please notify me.